ivy/functional/ivy/experimental/statistical.py

```diff
@@ -253,6 +253,32 @@ def nanmean(
     )
 
 
+@handle_out_argument
+@handle_nestable
+@handle_backend_invalid
+@handle_exceptions
+@to_native_arrays_and_back
+@handle_device
+def nanmin(
+    a: ivy.Array,
+    /,
+    *,
+    axis: Optional[Union[Tuple[int], int]] = None,
+    keepdims: Optional[bool] = False,
+    out: Optional[ivy.Array] = None,
+    initial: Optional[Union[int, float, complex]] = None,
+    where: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    return ivy.current_backend(a).nanmin(
```

Comment:
Doc Strings for documentation here as well 😄

Reply:
this is done